assets/sass/components/surveys/_googlesitekit-surveys-modal.scss

```diff
@@ -24,7 +24,7 @@
 
 	// The max-width of the survey element is 400px, so we allow it to break
 	// away from being "centered" after 400px.
-	@media (min-width: 400px) {
+	@media (min-width: 448px) {
```

Comment:
What's the reasoning behind `448px` here? I notice some of the Google Material breakpoints are around `450px`… maybe it'd be better to use that and be consistent with their breakpoints if we can't use our own?

![2021-06-29 11 50 35](https://user-images.githubusercontent.com/90871/123785429-6e48e400-d8d0-11eb-90db-0142774eb23b.gif)

Actually, we have one named `$width-xsmall`. Let's use that instead
```suggestion
	@media (min-width: $width-xsmall) {
```

Reply:
👍